packages/core/upload/admin/src/components/UploadAssetDialog/UploadAssetDialog.jsx

```diff
@@ -91,18 +94,33 @@ export const UploadAssetDialog = ({
     setAssets(nextAssets);
   };
 
+  console.log({
```

Comment:
Did QA locally and it's working but can you remove these console.logs and check the prettier error? 👀

Reply:
Ha yes sorry I forgot to let the PR in draft I'm pushing more fix and cleaning up :)